src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
The `href` attribute is linking to a SCSS file (`main.scss`). Browsers do not understand SCSS directly; it needs to be compiled to CSS first. Ensure that you compile your SCSS to CSS and link to the resulting CSS file instead.